YAML/YAML.sublime-syntax

```diff
@@ -51,11 +51,11 @@ variables:
   c_indicator: '[-?:,\[\]{}#&*!|>''"%@`]'
   c_flow_indicator: '[\[\]{},]'
   ns_word_char: '[0-9A-Za-z\-]'
-  ns_uri_char: '(?x: %\p{XDigit}{2} | [0-9A-Za-z\-#;/?:@&=+$,_.!~*''()\[\]] )'
+  ns_uri_char: '(?x: %[0-9A-Fa-f]{2} | [0-9A-Za-z\-#;/?:@&=+$,_.!~*''()\[\]] )'
```

Comment:
This change was motivated by Rust's engine not having `\p{XDigit}`. Java's engine for example does have `\p{XDigit}`, but it includes Unicode full-width and other digits (which does not correspond to YAML spec).

`\h` is also tricky because it is valid syntax in some engines but means horizontal whitespace. But given that it's used in other grammars already, we could have use that too, yeah.